native_client/ctcdecode/setup.py

```diff
@@ -56,12 +56,7 @@ def read(fname):
 
 decoder_module = Extension(
     name='ds_ctcdecoder._swigwrapper',
-    sources=['swigwrapper.i',
-             'ctc_beam_search_decoder.cpp',
-             'scorer.cpp',
-             'path_trie.cpp',
-             'decoder_utils.cpp',
-             'workspace_status.cc'],
```

Comment:
This is done on purpose as a compromise between developer speed of iteration and faster builds. If you move everything into common.a you have to rebuild the world on every change to these files.

Reply:
> This is done on purpose as a compromise between developer speed of iteration and faster builds. If you move everything into common.a you have to rebuild the world on every change to these files.

Yes, but on ctc decoder build, I can measure ~5 min win thanks to that because we can parallelize them this way.

Maybe I can put them into a second `.a` unit ?

Reply:
5 minutes?! D:

Reply:
Yeah, full task completing in 19 min instead of 24 min.